molecule/cookiecutter/scenario/driver/lxc/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/INSTALL.rst

```diff
@@ -6,14 +6,17 @@ Requirements
 ============
 
 * LXC
-* lxc-python2
 
 Install
```

Comment:
It's worth mentioning that py3 is currently not supported and extra build deps are required

Reply:
:+1: